src/Components/Util/useThingsError.js

```diff
@@ -0,0 +1,89 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import Link from '@material-ui/core/Link';
+
+import {ERRORS} from '../../Constants/Errors';
+
+
+const parseErrorMsg = (errMsg) => {
+    let errObj = {
+        errMsg: errMsg,
+        errCode: '',
+        errLinks: [],
+        errLeftover: ''
+    };
+
+    let reErrCode = /\(([-][0-9]*)\)/gi;
```

Comment:
both `reErrCode` and `reLink` may be defined as `const`?

Reply:
Updated in 43b8a4a.